admin_manual/configuration_server/antivirus_configuration.rst

```diff
@@ -5,211 +5,199 @@ Virus Scanner Support
 
-**Daemon (Socket)** 
+Configuring ClamAV within ownCloud
+----------------------------------
 
-In this mode, ClamAV runs in the background on the same server as the ownCloud
-installation. When there is no activity ``clamd`` places a minimal load on your
-system. However, if your users upload large volumes of files, you will see high
-CPU usage. So, please keep this in mind.
+Next, go to your ownCloud Admin page and set your ownCloud logging level to Everything.
 
-**Daemon Via Host:Port**
+.. figure:: ../images/antivirus-logging.png
 
-In this mode, ClamAv runs on a different server. This is a good option for
-ownCloud servers with high volumes of file uploads.
+Now, navigate to ``Settings -> Admin -> Additional``, where you’ll find the
+"**Antivirus Configuration**" panel.
+There, as below, you’ll see the configuration options which ownCloud will pass
+to ClamAV. 
 
-**Executable** 
+.. figure:: ../images/antivirus-config.png
 
-In this mode, ClamAV runs on the same server as the ownCloud installation, and
-the ``clamscan`` command only runs when a file is uploaded. ``clamscan`` is slow
-and not always reliable for on-demand usage; it is better to use one of the
-daemon modes.
+Mode Configuration
+^^^^^^^^^^^^^^^^^^
+
+ClamAV runs in one of three modes: `Daemon (Socket)`_, `Daemon`_, and `Executable`_.
 
 Daemon (Socket)
 ^^^^^^^^^^^^^^^
 
-ownCloud should detect your ``clamd`` socket and fill in the ``Socket``
-field. This is the ``LocalSocket`` option in ``clamd.conf``. You can
+In this mode, ClamAV runs in the background on the same server as the ownCloud installation. 
+When there is no activity ``clamd`` places a minimal load on your
+system. 
+However, if your users upload large volumes of files, you will see high CPU usage. 
+Please keep this in mind.
+
+ownCloud should detect your ``clamd`` socket and fill in the ``Socket`` field. 
+This is the ``LocalSocket`` option in ``clamd.conf``. You can
 run ``netstat`` to verify::
 
    netstat -a|grep clam
    unix 2 [ ACC ] STREAM LISTENING 15857 /var/run/clamav/clamd.ctl
 
   .. figure:: ../images/antivirus-daemon-socket.png
 
-The ``Stream Length`` value sets the number of bytes read in one pass.
-10485760 bytes, or ten megabytes, is the default. This value should be 
-no larger than the PHP ``memory_limit`` settings, or physical memory if 
-``memory_limit`` is set to -1 (no limit).
+The ``Stream Length`` value sets the number of bytes to read in one pass.
+10485760 bytes, or ten megabytes, is the default. 
+This value should be no larger than the PHP ``memory_limit`` settings or physical memory if ``memory_limit`` is set to -1 (no limit).
 
-``Action for infected files found while scanning`` gives you the choice of
-logging any alerts without deleting the files, or immediately deleting
-infected files.
+``Action for infected files found while scanning`` gives you the choice of logging any alerts without deleting the files or immediately deleting infected files.
 
```

Comment:
I'm having trouble with a local install and don't see the anti-virus app when running `./occ app:list`. Not sure if I'm missing something. But given its not there, I can't do due diligence on this section of the file.